js/main.js

```diff
@@ -0,0 +1,30 @@
+var $photoUrl = document.querySelector('#photo-url');
+var $photoOutput = document.querySelector('img');
```

Comment:
This will probably break in the future, this app will require more `img` elements in the future.